diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtilsTest.kt

```diff
@@ -746,6 +746,7 @@ private class PrettyPrintingVisitor(private val elementType: IElementType,
             )
         }
     }
+    @Suppress("WRONG_WHITESPACE")
```

Comment:
Why is it wrong whitespace? Do we support such cases at all? I remember that we had issues regarding support for empty lambdas

Reply:
Because now it fails with `NPE`. But I fixed this bug in this PR.

Reply:
So it fails only for `diktat from release` workflow, that's fine. You can unsuppress it right away